pkg/postage/mock/service.go

```diff
@@ -114,7 +114,9 @@ func (m *mockPostage) HandleTopUp(batchID []byte, amount *big.Int) {
 	}
 }
 
-func (m *mockPostage) HandleDepthIncrease(_ []byte, _ uint8) {}
+func (m *mockPostage) HandleDepthIncrease(_ []byte, _ uint8) {
+	panic("not implemented")
```

Comment:
So I think we decided to return error on the API. This is not the same as putting a panic here.

You need to change the `devnode.go` file, the mockcontract to return error on these things.